metrics-observer-prometheus/src/lib.rs

```diff
@@ -28,6 +35,23 @@ impl PrometheusBuilder {
         self.quantiles = parse_quantiles(quantiles);
         self
     }
+
+    /// Sets the buckets to use when rendering summaries.
+    ///
+    /// Buckets values represent the higher bound of each buckets.
+    pub fn set_buckets(mut self, values: &[u64]) -> Self {
+        self.buckets = values.to_vec();
+        self
+    }
+
+    /// Sets the buckets for a specific metric, overidding the default.
+    ///
+    /// Matches the metric name using `ends_with`.
```

Comment:
Can you extend this comment to mention that it not only matches the suffix, but searches for the longest match first?